Gemfile

```diff
@@ -107,9 +103,9 @@ gem 'rails-html-sanitizer', '~> 1.0.4'
 gem 'sqlite3'#,  '1.3.10'
 
 # Use Unicorn as the web server
-# FIXME: Switch to Puma for Rails 5
-gem 'unicorn',  '4.9.0', group: :production
-
+gem 'unicorn',  '4.9.0'
```

Comment:
are we using an outdated version of `unicorn`? why?

Reply:
yes, the most recent version of `unicorn` is `5.4.0`.  But `4.9.0.` is what we were using already so I didn't bother upgrading it (the only change here is that `unicorn` is no longer restricted to the production env only